### DISCUSSION TITLE
How to actually use the bot

### DISCUSSION THREAD

**Discussion by psychosymbol** [NoMeansNowastaken/KarutaSniper#8] [Q&A]
I'm don't know much about python, and this is my first-time using python script file so i have a question.
I've installed everything in the requirement and added the pytesseract to my PATH.
When I try to run main.py it said i need to find some token (y/n) so I type in y and enter, it pop out some text and close by it self really fast so I didn't catch what it said and I'm stuck here. Where did I go wrong?

**Comment by splat58** [comment]
Currently, you can only see it if ran from cmd, I just updated so you should be able to run it now without using cmd

**Comment by psychosymbol** [comment]
I just downloaded the latest version now and it work now, I got token from main.py so I edit the config file then run it again then it gave me this error (attach in the pic) before closing itself. I check with Discord Token Finder and the token it gave me is the same one as the one I get from main.py.
![image](https://user-images.githubusercontent.com/72127164/216176230-30549e32-134d-4d97-aa2a-4a3d41057e32.png)

**Comment by splat58** [comment]
try to get your discord token with a method found off google as the one i used is pretty outdated and might not work anymore
dont get ratted tho (be careful when running files)
